src/tests/ftest/control/config_generate_output.yaml

```diff
@@ -5,11 +5,8 @@ server_config:
   engines_per_host: 1
   engines:
     0:
-      storage:
-        0:
-          class: ram
-          scm_mount: /mnt/daos
-          scm_size: 16
+      storage: auto
+  system_ram_reserved: 2
```

Comment:
Do we need to reserve ram on tests targeted to run on HW Large clusters?

Reply:
no, updated